website/docs/r/data_protection_backup_policy_postgresql.html.markdown

```diff
@@ -111,6 +125,24 @@ A `criteria` block supports the following:
 
 * `weeks_of_month` - (Optional) Possible values are `First`, `Second`, `Third`, `Fourth` and `Last`. Changing this forces a new Backup Policy PostgreSQL to be created.
 
+---
+
+A `life_cycle` block supports the following:
+
+* `data_store_type` - (Required) The type of data store. Possible values are `VaultStore`, `ArchiveStore`. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `duration` - (Required) Duration after which the backup is deleted. It should follow `ISO 8601` duration format. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+* `target_copy_setting` - (Optional) A `target_copy_setting` block as defined below. Changing this forces a new Backup Policy PostgreSQL to be created.
+
+---
+
+A `target_copy_setting` block supports the following:
+
+* `copy_option` - (Required) Specifies when the backups are tiered across two or more selected data stores as a json encoded string. Changing this forces a new Backup Policy PostgreSQL to be created.
```

Comment:
and this might be better as 
```suggestion
* `option_json` - (Required) Specifies when the backups are tiered across two or more selected data stores as a json encoded string. Changing this forces a new Backup Policy PostgreSQL to be created.
```
?

Reply:
Fixed.